src/TLuaInterpreter.cpp

```diff
@@ -5869,6 +5869,39 @@ int TLuaInterpreter::tempButton(lua_State* L)
     return 1;
 }
 
+int TLuaInterpreter::setButtonStyleSheet(lua_State* L)
+{
+    //args: name, css text
+    QString name, css;
+
+    if (!lua_isstring(L, 1)) {
+        lua_pushfstring(L, "setButtonStyleSheet: bad argument #1 type (name as string expected, got %s!)", luaL_typename(L, 1));
+        return lua_error(L);
+    } else {
+        name = QString::fromUtf8(lua_tostring(L, 1));
+    }
+    if (!lua_isstring(L, 2)) {
+        lua_pushfstring(L, "setButtonStyleSheet: bad argument #2 type (css as string expected, got %s!)", luaL_typename(L, 2));
+        return lua_error(L);
+    } else {
+        css = QString::fromUtf8(lua_tostring(L, 2));
+    }
+
+    Host& host = getHostFromLua(L);
+    auto actionsList = host.getActionUnit()->findActionsByName(name);
+    if (actionsList.empty()) {
+        lua_pushnil(L);
+        lua_pushfstring(L, "setButtonStyleSheet: no button named \"%s\" found", name.toUtf8().constData());
```

Comment:
As a _run-time_ error (**nil+error message**) we are moving over to a message of the form:

    lua_pushfstring(L, "No button named \"%s\" found.", name.toUtf8().constData());

Also, this particular aspect of the `TAction` class (which I wish we had called `TButton` to avoid confusing them with `QAction` which is related but significantly different)  - the _StyleSheet_, applies also to __Toolbars__ (top level folder/container/collections of `TAction` instances) and __Menus__ (2nd and lower levels of folder/container/collections of `TAction` instances) - this is important because one of the 'C's in CCS is _Cascading_ and settings in Toolbars and Menus can affect the Menu and Button items they can contain!

Reply:
Yep, true!